docs/run-keep-ecdsa.adoc

```diff
@@ -323,6 +323,65 @@ reference].
 |No
 |===
 
+==== Example BeneficiaryAddresses
+
+[%header,cols=2*]
+|===
+|Description
+|Address
+
+|mainnet xpub extended public key
+|`BeneficiaryAddress = "xpub6Cg41S21VrxkW1WBTZJn95KNpHozP2Xc6AhG27ZcvZvH8XyNzunEqLdk9dxyXQUoy7ALWQFNn5K1me74aEMtS6pUgNDuCYTTMsJzCAk9sk1"`
```

Comment:
What do you think about breaking the data a little bit, but remaining a good example? This would help to ensure the samples don't get copied and used in a client by any chance. 
We could add something in the middle for each case, e.g. `xpub6Cg41S21VrxkW1____REPLACE_WITH_VALID_DATA____XyNzunEqLdk9dxyXQUoy7ALWQFNn5K1me74aEMtS6pUgNDuCYTTMsJzCAk9sk1`.

Reply:
I like it!